x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/mapper/SemanticTextFieldTests.java

```diff
@@ -204,6 +214,22 @@ public static SemanticTextField randomSemanticText(String fieldName, Model model
         );
     }
 
+    /**
+     * Returns a randomly generated object for Semantic Text tests purpose.
+     */
+    public static Object randomSemanticTextInput() {
+        int randomInt = randomIntBetween(0, 5);
+        return switch (randomInt) {
+            case 0 -> randomAlphaOfLengthBetween(10, 20);
```

Comment:
Nit - Maybe we should have less priority for using boolean / numbers via `rarely()`? Something like

```java
if (rarely) {
    return switch(randomIntBetween(0, 4)) {
            case 0 -> randomInt();
            case 1 -> randomLong();
            case 2 -> randomFloat();
            case 3 -> randomBoolean();
            case 4 -> randomDouble();
   }
} else {
   return randomAlphaOfLengthBetween(10, 20);
}
```